internal_ws/ykcompile/src/lib.rs

```diff
@@ -298,8 +298,9 @@ pub extern "sysv64" fn bh_push_vec(
 ) {
     let fname =
         unsafe { std::str::from_utf8(std::slice::from_raw_parts(sym_ptr, sym_len)).unwrap() };
+    let body = SIR.body(fname).unwrap();
     let fi = FrameInfo {
-        sym: fname.to_string(),
+        body,
```

Comment:
I'm not quite sure if we need to clone the Arc here, since due to the boxing below I think the borrow checker wouldn't catch it if we do. Though since `SIR.body` already creates a clone, I think this is fine. Can't hurt to double check with @vext01 though.

Reply:
`Arc<Body>` has an `'static` lifetime, so this is fine.